mvp/resources/README

```diff
@@ -1 +1,71 @@
+
+This zip file contains:                                                                                                                             
+    - An Eclipse directory containing the Galasa plug-in                                                                                                
+    - A Maven directory containing dependencies that are required for building Galasa tests                                                             
+    - A Javadoc directory containing Javadoc API documentation for the Galasa Managers                                                                                                                              
+    - An isolated.tar file - an optional Docker image that hosts the Eclipse and Maven directories. Use this file to host Galasa                        
+    on an internal server that can be accessed by other users.                                                                                          
+    - A docs.jar file that enables you to run the Galasa website locally on your machine                                                                
+    - A Notices file                                                                                                                                    
+    - Various language files                                                                                                                            
+                                                                                                 
+                                                                                                                                                    
+You can find out more about Galasa on the Galasa website https://galasa.dev, or you can host the website locally by 
+running the `docs.jar` file that is contained in this zip file on your machine or on an internal server.                                                                
+                                                                                                                                                 
+                                                                                                                                                    
+Running the Galasa website locally                                                                                                                    
+-----------------------------------                                                                                                                         
+                                     
+You can run the `docs.jar` file locally by using the standard java command with the -jar option. 
+
+Running the website on your local machine
+-------------------------------------------
+
+From a command line, run the following command in the directory into which you extracted the `docs.jar` file: 
```

Comment:
because a jar file is an archive we should be clear this is the directory in which you extracted the download (which contained docs.jar) not where the docs were extracted

Reply:
updated